### ISSUE TITLE
MathJax issue ?

### ISSUE THREAD

**Issue by damien-git** [dequelabs/axe-core#1685]
aXe seems to have a hard time analyzing https://users.math.msu.edu/users/wongwwy/expository/torsion.html . I noticed the issue first with axe-webdriverjs. I also tried the aXe plugin in both Firefox and Chromium, and had to force stop it in both cases.
Possibly because of the use of MathJax ?

**Comment by straker** [comment]
Looking at the page, it seems the color-contrast rule is taking 23 seconds to complete. Axe-core's color-contrast rule is one of the slowest rules we have, and it looks like all the individual math symbols are being checked for color contrast.

**Comment by damien-git** [comment]
Interesting. Would there be any way to accelerate the process for similar pages ?

**Comment by straker** [comment]
Right now our only workaround is to turn off the color-contrast rule for such pages.

```js
axe.run({
  "rules": {
    "color-contrast": { enabled: false },
  }
});
```

**Comment by WilcoFiers** [comment]
It is also worth noting that we're working on a major performance improvement to the color-contrast rule. That will hopefully help to solve this.

**Comment by damien-git** [comment]
This is good news.
I wrote a [domain accessibility audit](https://github.com/MSU-NatSci/DomainAccessibilityAudit) web app based on axe that can check many websites at once and report statistics. Improved axe-core performance would be a must for that.

**Comment by jeankaplansky** [comment]
ID'd as color-contrast issue. We're going to make one entry into the release notes for all color-contrast fixes/enhancements.

**Comment by jeankaplansky** [comment]
Added to release notes as of 2/04/2019

**Comment by padmavemulapati** [comment]
As the reported url no longer working, verified on the another suggested url, `http://web.archive.org/web/20180625151005/https://users.math.msu.edu/users/wongwwy/expository/torsion.html ` time taking here is less than 2secs
![image](https://user-images.githubusercontent.com/47409579/73907923-0efac600-48ce-11ea-947a-5b983c820ee3.png)
